README.md

```diff
@@ -119,6 +122,16 @@ The set of attributes the user has configured for the transaction.
 #### YotiClient
 
 Allows your app to retrieve a user profile, given an encrypted token.
+
+## How to run the examples
+
+The examples can be found in the [examples folder](https://github.com/getyoti/php/tree/master/example) which requires few setups to work as explained below.
+
+- Create your application in Yoti Dashboard (this requires having a Yoti account)
+- Fill in YOTI_APPLICATION_ID and YOTI_APPLICATION_SCENARIO_ID inside index.html file 
```

Comment:
Fill in the YOTI_APPLICATION_ID and YOTI_APPLICATION_SCENARIO_ID values in the [index.html](example/index.html) file